loads/src/main/java/org/mousephenotype/cda/loads/cdaloader/configs/ConfigBeans.java

```diff
@@ -0,0 +1,395 @@
+/*******************************************************************************
+ * Copyright © 2015 EMBL - European Bioinformatics Institute
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ ******************************************************************************/
+
+package org.mousephenotype.cda.loads.cdaloader.configs;
+
+import org.mousephenotype.cda.db.pojo.Allele;
+import org.mousephenotype.cda.db.pojo.GenomicFeature;
+import org.mousephenotype.cda.db.pojo.OntologyTerm;
+import org.mousephenotype.cda.db.pojo.Strain;
+import org.mousephenotype.cda.enumerations.DbIdType;
+import org.mousephenotype.cda.loads.cdaloader.exceptions.CdaLoaderException;
+import org.mousephenotype.cda.loads.cdaloader.steps.*;
+import org.mousephenotype.cda.loads.cdaloader.support.CdaLoaderUtils;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by mrelac on 03/05/16.
+ */
+@Configuration
+public class ConfigBeans {
+
+    private Map<String, OntologyTerm>   mgiFeatureTypes;
+
+    private Map<String, Allele>         alleles         = new HashMap<>(150000);    // key = allele accession id
```

Comment:
do you really need to set the initial capacity of these?  These maps will not grow large enough to really need that many buckets.  Recommend using the defaults (rehashing is fast enough?)